.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
```

Comment:
This should be 4 and 6 and then travis will download the latest of those versions.